CHANGELOG.md

```diff
@@ -14,6 +14,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * Fixed
   * None
 
+## 6.4.3 (2023-12-13)
+
+* Breaking Changes
+  * None
+* Added
+  * None
+* Fixed
+  * [#768](https://github.com/binarylogic/authlogic/issues/768) - Fixed
+    compatibility issues with Rails 7.1
```

Comment:
```suggestion
  * [#769](https://github.com/binarylogic/authlogic/issues/769) - Fixed GH workflow 
```

Reply:
Hi @tiegz, thank you for merging the #767 I was really looking forward to it. Anyway, I have made the requested changes to my PR. Please let me know if anything else is needed, I will follow up the same or next day